chrome/locale/en-US/zoteroocr.dtd

```diff
@@ -1,2 +0,0 @@
-<!ENTITY ocr.menu.label			"OCR selected PDF(s)">
```

Comment:
Please don't remove this file. It is required for a plugin which supports both Zotero 6 and Zotero 7.